package.json

```diff
@@ -1,19 +1,21 @@
 {
   "name": "jest-vue-preprocessor",
-  "version": "0.2.0",
+  "version": "0.2.0-development",
   "description": "Preprocessor that allows importing of .vue files in jest tests",
   "engines": {
     "node": ">= 6.3"
   },
   "main": "index.js",
   "scripts": {
+    "commit": "git-cz",
     "lint": "eslint .",
     "test": "jest --no-cache",
-    "test:ci": "jest --coverage"
+    "test:ci": "jest --coverage",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
```

Comment:
We need to add following:
- git hooks via `husky` commit message, prepush
- commit validation, because not everyone is reading contributing guidelines ( unfortunately ) and users of IDE like webstorm et all are not using CLI for commits.

Pls take a look at https://github.com/Hotell/typescript-lib-starter/blob/master/package.json#L35